docs/TTS/GetCapabilities/index.md

```diff
@@ -25,6 +25,33 @@ Purpose
 |speechCapabilities|[Common.SpeechCapabilities](../../common/enums/index.md#speechcapabilities)|true|array: true<br>minsize: 1<br>maxsize: 5|
 |prerecordedSpeechCapabilities|[Common.PrerecordedSpeech](../../common/enums/index.md#prerecordedspeech)|true|array: true<br>minsize: 1<br>maxsize: 5|
 
+!!! NOTE
+
+  Description of possible SpeechCapabilities
+  TTS is performed by the HMI through:
+  
+  1. TEXT - Plain text format. TTSChunks with this type provide plain text to be spoken in the `text` field
+  2. SAPI_PHONEMES - Microsoft Speech API Format. TTSChunks with this type provide a group of phonemes with this format in the `text` field
+  3. LHPLUS_PHONEMES - LH+ Phoneme format. TTSChunks with this type provide a group of phonemes with this format in the `text` field
+  4. PRE_RECORDED - Prerecorded sounds. TTSChunks with this type provide a value from [Common.PrerecordedSpeech](../../common/enums/index.md#prerecordedspeech) in the `text` field
+  5. SILENCE - Definite amount of time. TTSChunks with this type provide amount of time HMI must keep silence.
+  6. FILE - Uploaded audio files. TTSChunks with this type provide a filename pointing to an audio file previously uploaded through a PutFile RPC in the `text` field
+      * These audio files can be in one of the following file types/specifications, all must be supported in order for the HMI to support the `FILE` option:
+          * AUDIO_WAVE
```

Comment:
is there a time limit for the audio content? for example 10 or 30 seconds?

Reply:
No limit was given in the proposal, so no limit is implemented as of now